service-manual/the-team/learning-and-development/foundation-day.md

```diff
@@ -87,5 +87,5 @@ As preparation before the day, we ask that participants:
 
 To request a place on the foundation day, please fill in this [Civil Service Learning request form](https://civilservicelearning.civilservice.gov.uk/learning-opportunities/digital-foundation-day)
 
-Any questions? Please contact the GDS Skills Team: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:service-manager-programme@digital.cabinet-office.gov.uk)
+Any questions? Please contact: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:digital.academy@dwp.gsi.gov.uk)
```

Comment:
Isobel,

Apologies, I have not done this before.

I want the emails to go to the DWP email address, so can this stay the same?

Thanks,

Claire



Claire Moles ¦ Recruitment & Skills Team ¦ Government Digital Service ¦
Cabinet Office
7th Floor, Aviation House, 125 Kingsway, London, WC2B 6NH

Tel: 07717715727

On 20 August 2015 at 10:30, Isabell Long <notifications@github.com> wrote:

> In service-manual/the-team/learning-and-development/foundation-day.md
> <https://github.com/alphagov/government-service-design-manual/pull/586#discussion_r37510869>
> :
>
> > @@ -87,5 +87,5 @@ As preparation before the day, we ask that participants:
> >
> >  To request a place on the foundation day, please fill in this [Civil Service Learning request form](https://civilservicelearning.civilservice.gov.uk/learning-opportunities/digital-foundation-day)
> >
> > -Any questions? Please contact the GDS Skills Team: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:service-manager-programme@digital.cabinet-office.gov.uk)
> > +Any questions? Please contact: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:digital.academy@dwp.gsi.gov.uk)
>
> Thanks for this! A quick thing though: you'll need to change the text
> between the [] that currently advertises a CO address, as the actual
> email will go to a DWP address.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/government-service-design-manual/pull/586/files#r37510869>
> .
>